react-components/src/hooks/use3dScenes.tsx

```diff
@@ -33,6 +43,9 @@ type Use3dScenesQueryResult = {
   scene360Collections: Array<
     EdgeItem<Record<string, Record<string, Cdf3dImage360CollectionProperties>>>
   >;
+  sceneGroundPlanes: Array<NodeItem<GroundPlaneProperties>>;
+  sceneGroundPlaneEdges: Array<EdgeItem<Record<string, Record<string, Transformation3d>>>>;
+  sceneSkybox: Array<NodeItem<SkyboxProperties>>;
```

Comment:
We are returning a lot of arrays here. Is this only for internal use? If it's also for external use, we should probably have one list with this info per scene

Reply:
`Use3dScenesQueryResult` is only for internal use in the `use3dScenes` hook